items.json

```diff
@@ -4848,9 +4848,9 @@
     "name": "Fine Steel Cavalry Broadsword",
     "culture": "Battania",
     "type": "OneHandedWeapon",
-    "price": 7545,
-    "weight": 1.55,
-    "tier": 10.0334482,
+    "price": 11076,
+    "weight": 1.44,
+    "tier": 12.394105,

```

Comment:
I did not make updates to the Fine Steel Cavalry Broadsword so I'm unsure why this appears here.